proto/rill/runtime/v1/queries.proto

```diff
@@ -153,6 +153,13 @@ service QueryService {
     };
   }
 
+  rpc MetricsViewResolveTimeRanges(MetricsViewResolveTimeRangesRequest) returns (MetricsViewResolveTimeRangesResponse) {
```

Comment:
Can we call this `MetricsViewTimeRanges` – to align with format of the other `MetricsViewXXX` API names